hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/RocksDBUtils.java

```diff
@@ -45,4 +52,28 @@ public static List<ColumnFamilyDescriptor> getColumnFamilyDescriptors(
     }
     return cfs;
   }
+
+  public static ColumnFamilyHandle getColumnFamilyHandle(String columnFamilyName, List<ColumnFamilyHandle> cfHandleList)
+      throws RocksDBException {
+    byte[] nameBytes = columnFamilyName.getBytes(StandardCharsets.UTF_8);
+
+    for (ColumnFamilyHandle cf : cfHandleList) {
+      if (Arrays.equals(cf.getName(), nameBytes)) {
+        return cf;
+      }
+    }
+
+    return null;
+  }
+
+  public static <T> T getColumnFamilyValue(ManagedRocksDB db,
```

Comment:
nit:
1. `getValueFromColumnFamily` or just `getValue`. `getColumnFamilyValue` feels like goona return the `columnFamily` information and not the value for the key.
2. I think alignment is off in line number 70.

Reply:
yeah 'getValue' makes sense! and line 70 alignment fixed!